js/components/frame.js

```diff
@@ -337,8 +337,7 @@ class Frame extends ImmutableComponent {
           this.props.frame)
         const key = this.props.frame.get('key')
         const parsedUrl = urlParse(event.url)
-        if (['http:', 'https:', 'about:', 'chrome:', 'chrome-extension:', 'file:',
-             'view-source:'].includes(parsedUrl.protocol)) {
+        if (parsedUrl.protocol) {
```

Comment:
I'm not sure if this check is really necessary. Is there a circumstance under which we would not want to update the url?